src/locales/en.json

```diff
@@ -982,7 +982,9 @@
         "FAILED_TO_FINALIZE": "The inscription transaction failed to finalize. Please try again or contact support.",
         "SERVER_ERROR": "An unknown server error occurred. Please try again or contact support."
       },
-      "TOO_MANY_REPEATS": "You can only create up to 24 inscriptions in a single request"
+      "TOO_MANY_REPEATS": "You can only create up to 24 inscriptions in a single request",
```

Comment:
I think it would be good to move the `24` number to some constant (in a shared file or the component where it's used) to not change the translations in case the number needs to be changed, right? We can then pass it like this:
<img width="778" alt="Monosnap index tsx — xverse-web-extension 2023-12-14 14-30-53" src="https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/d4009e9a-9a0a-4a77-9d5d-1e5370ac7c3a">

Reply:
That's a valid point. Although this wasn't included in the current pull request, I've addressed it anyway 😄